### DISCUSSION TITLE
Light-Portal/getting-started/updating

### DISCUSSION THREAD

**Discussion by giscus** [dragomano/Light-Portal#244] [Q&A]
# Light-Portal/getting-started/updating

Information on how to update the portal

https://dragomano.github.io/Light-Portal/getting-started/updating

**Comment by dragomano** [comment]
Note for those having trouble updating from **2.6.*** to **2.7.***: In the old versions, there was a Todays plugin, but it’s been removed in the new ones. So, if you have that folder, make sure to delete the `Sources/LightPortal/Addons/Todays` folder before updating. If you run into a 500 error, first rename the `Sources/LightPortal/Addons` folder to check if the error is related to outdated code from any plugin. If that doesn’t help and the error keeps happening, rename Sources/LightPortal to get the forum working again and send me the error details.
